src/Gladiators/Entities/Gladiator.cs

```diff
@@ -6,5 +6,60 @@ namespace Gladiators.Entities
 {
     public class Gladiator
     {
+        public Gladiator(string name, uint healthPoints = 50, uint attack = 10)
+        {
+            if (name.Length < 3 || name.Length > 15) throw new InvalidNameException();
+
```

Comment:
Il manque les validations pour healthpoints et attack non ? 
 - - HealthPoints | integer,  must always be greater or equal to zero.
 - - Attack | integer,  must always be greater or equal to zero.

Reply:
C'est deja des `uint`